requirements-metrics.txt

```diff
@@ -0,0 +1,3 @@
+numpy
+pystan
+fbprophet
```

Comment:
We should remove them from `requirements-base.txt`, which are commented at the moment.